bin/pygrb/pycbc_pygrb_pp_workflow

```diff
@@ -449,17 +449,19 @@ _workflow.makedir(out_dir)
 
 # Offtrials and injection sets requested by the user
 num_trials = int(wflow.cp.get('trig_combiner', 'num-trials'))
-offtrials = ["offtrial_%s" % (i+1) for i in range(num_trials)]
+offtrials = [f"offtrial_{i+1}" for i in range(num_trials)]
+
+# Sensitivity plots of each injection set
 out_dir = rdir['exclusion_distances']
 _workflow.makedir(out_dir)
 for i, offtrial in enumerate(offtrials):
     out_dir = rdir[f'exclusion_distances/{offtrial}']
     _workflow.makedir(out_dir)
     files = _workflow.FileList([])
     for inj_set in inj_sets:
-        inj_file = list(filter(lambda x: inj_set.lower() in x.lower(), inj_files))
-        inj_file = inj_file[0] if len(inj_file) == 1 else None
-        tags = [offtrial, inj_set]
+        # Retrieve the injections result File for the inj_set label
+        inj_file = inj_files.find_output_with_tag(inj_set)[0]
```

Comment:
Then this also has `fail_if_not_single_file=True` and we don't duplicate the sanity check.